### PR TITLE
Landuse transportation fix

### PR DESCRIPTION
**Pull request by thesocialdev**
1) tweak imposm configuration:  Remove boundary from linear_tags config. It prevents planet_osm_polygon_landuse to load some data such as natural_reserver
2) fix call to labelgrid function
3)  Fix layer_transportation_name_to_class. Despite the data is generated as MVT, it's not rendered by mapnik.
4)  Minor nits: "is" column, max zoom = 15, remove extra params and reorder the layers
5)  Vendor postgis-vt-utils and include in the etl documentation

### CODE REVIEW

**Comment by thesocialdev** [comment]
@ARolek here are some fixes that I did regarding mapnik renderer.

File: sql/functions/layer_transportation.sql [hunk 1]
```diff
@@ -1,35 +1,30 @@
...
     SELECT
         osm_id,
         way,
-        CASE
-          WHEN highway IS NOT NULL THEN layer_transportation_name_to_class(highway, osm_id)
-          WHEN railway IS NOT NULL THEN layer_transportation_name_to_class(railway, osm_id)
-          WHEN access IS NOT NULL THEN layer_transportation_name_to_class(access, osm_id)
-          ELSE bail_out('Unexpected layer_transportation road row with osm_id=%s', osm_id::TEXT)
-        END AS class,
+        layer_transportation_name_to_class(highway, railway, access, osm_id) AS class,
```
Comment (ARolek) [inline sql/functions/layer_transportation.sql:19]: Nice. I like this imporvement! I felt the old way was a wart. Thanks for the fix!

**Review by ARolek — APPROVED** [review]
Excellent PR! You squashed a ton of bugs and added some great polish. Thank you! 👏

File: sql/functions/layer_transportation.sql (hunk 1 above)
Comment (thesocialdev) [reply sql/functions/layer_transportation.sql:19]: Thanks!

**Pull request merged**